docs/source/release_notes/release-notes.rst

```diff
@@ -36,4 +36,13 @@ v0.4.1
     *   - |improved|
         - We now have continuously updated release notes
 
+    *   - |improved|
+        - Cleaned Atlas CLI code
+
+    *   - |fixed|
+        - C* 2.x schema mutation bug `431 <https://github.com/palantir/atlasdb/issues/431>`_ (`444 <https://github.com/palantir/atlasdb/issues/444>`_ not yet fixed)
+
+    *   - |fixed|
```

Comment:
I think maybe we want to put all of these issues underneath "done in 4.1" and have a new section for 5.0 that only represents the current delta between 4.1 and 5.0